js/rendererjs/update.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import request from 'request'
+import semver from 'semver'
+import { shell } from 'electron'
+import UpdateComponent from './updatecomponent.js'
+
+const getReleases = (url) => new Promise((resolve, reject) => {
+	request({
+		url,
+		json: true,
+		headers: {
+			'User-Agent': 'Sia-UI',
```

Comment:
the github API requires that you set the user agent for all requests, otherwise it returns `403 Forbidden`